README.md

```diff
@@ -9,11 +9,11 @@ the COOL environment.
 
 - [Terraform](https://www.terraform.io/) installed on your system.
 - An accessible AWS S3 bucket to store Terraform state
-  (specified [here](backend.tf)).
+  (specified in [backend.tf](backend.tf)).
```

Comment:
I suggest putting these filenames in monospaced font.  That is allowed inside of link text.

Reply:
Thanks for the good suggestion- see https://github.com/cisagov/cool-assessment-terraform/pull/67/commits/1cac783a9991fa8b40a89a388dbdfd70f65d63b6.